packages/rolldown/src/utils/compose-js-plugins.ts

```diff
@@ -269,13 +257,7 @@ function isComposablePlugin(plugin: RolldownPlugin): plugin is Plugin {
     return false
   }
 
-  if ('resolveId' in plugin) {
-    // FIXME: Conflict with the `skip` option in `PluginContext#resolve`. Since we can't detect it in advance,
-    // we have to bailout all plugins with `resolveId` hook.
-    return false
-  }
-
-  if (Object.keys(plugin).some((k) => unsupportedHooks.has(k))) {
```

Comment:
Seems “some” would execute less times?

Reply:
Also need to consider the “name” property.

Reply:
Or, I can't